examples/web-instrumentation/browser/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "end-to-end-instrumentation-browser",
+  "version": "0.0.14",
+  "description": "Following https://webpack.js.org/guides/getting-started/",
+  "private": true,
+  "scripts": {
+    "prebuild": "npx eslint -c .eslintrc.js 'src/**/*.js' --fix",
+    "build": "webpack"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-google": "^0.14.0",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10"
+  },
+  "dependencies": {
+    "@opentelemetry/exporter-collector": "^0.3.2",
```

Comment:
It would be nice if you can switch to latest version `0.4.0`.

Reply:
Changed in new commit